docker-compose.yml

```diff
@@ -53,6 +53,11 @@ services:
       # EMAIL_HOST_PASSWORD: smtp_account_password
       # EMAIL_PORT: 587 # optional
       # EMAIL_USE_TLS: 'true' # optional
+      # Env vars for GitHub Integration
```

Comment:
I don't think these should go here - we want this to be a minimal file - email hints are useful as they are very helpful to set up the platform but we have dozens of other env vars that we don't add here.

Reply:
Okay, I removed it now.